### DISCUSSION TITLE
It is recommended to increase the pcap traffic capture

### DISCUSSION THREAD

**Discussion by rockmelodies** [projectdiscovery/nuclei#3245] [Ideas]
It is recommended to increase the pcap traffic capture

**Comment by ehsandeep** [comment]
@rockmelodies could you please clarify what you mean by "increase the pcap traffic capture" or what you looking to achieve with nuclei?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
